common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -104,6 +104,14 @@ def student_view(self, context):
 
         return fragment
 
+    def anonymous_view(self, context):
+        """
+        Renders the anonymous view of the block in the LMS.
+        """
+        context = context if context else {}
```

Comment:
You can just do `def anonymous_view(self, context={}):` instead of this.

Reply:
Actually that will not be needed after the next line is removed.

Reply:
👍